standup_notes/__main__.py

```diff
@@ -133,18 +182,13 @@ def delete_notes(date_to_delete):
     if files_to_delete:
         print("Here are the file to be deleted")
         print(*files_to_delete, sep='\n')
-        while True:
-            result = input("Are you sure you want to delete these files y/n: ")
-            if result.lower() == 'y':
-                for file in files_to_delete:
-                    os.remove(os.path.join(STANDUP_NOTES, file))
+        result = verify_input("Are you sure you want to delete these files y/n: ")
+        if result:
+            for file in files_to_delete:
+                os.remove(os.path.join(STANDUP_NOTES, file))
                 print("Files have been deleted.")
-                break
-            if result.lower() == 'n':
+            if not result:
```

Comment:
this will never get run, since it's in an if block where it checks if result is true.

Reply:
It was one block to high, moved it down one

Reply:
You can also just change it to "else"